### DISCUSSION TITLE
Adding/removing edges from relay connection

### DISCUSSION THREAD

**Discussion by pkrakesh** [g18e/ariadne-relay#18] [Q&A] [answered]
How can I add/remove edges from a relay connection so that I can perform optimistic updates in the fronted? I am new to Ariadne and schema first approach. I have had experience with Graphene before.

**Comment by markedwards** [comment, marked as answer]
There isn’t anything connection-specific about how data is mutated.  Connections and edges only provide a mechanism to query and paginate sets of data.

Mutations are covered in the main Ariadne docs: https://ariadnegraphql.org/docs/mutations

**Reply by pkrakesh** [reply]
Hi Mark, Thank you very much!
